s2fft/precompute_transforms/construct.py

```diff
@@ -53,27 +60,102 @@ def spin_spherical_kernel(
             "Reality acceleration only supports spin 0 fields. "
             + "Defering to complex transform."
         )
+    if mode.lower() == "price_mcewen" and abs(spin) >= 5:
```

Comment:
I'm not sure we should override the users selection.  If they select auto we're free to choose the recursion but otherwise we should follow what they set.  Users never read warnings, so I don't think they would realise the recusion has been changed.  

If we are in the setting where we know price-mcewen recursion is going to be unstable, then we should throw a wanring.  Or perhaps we should throw and error and simply stop execution?  What do you think we should generally do?

Reply:
@jasonmcewen this is the same behaviour we adopt for if the user provides reality=True for spin != 0. We warn them that this will cause errors, and then default to reality=False.

Ultimately we need to decide whether we should let the user call the functions in a way that we know for sure a priori will cause errors.

If we just warn the user and don't switch, they will get bad results and then spend time trying to figure it out and eventually switch to Risbo.

If we throw a ValueError pointing to Risbo they will switch to Risbo more quickly.

In either case, the user will have to switch to Risbo, so I don't see the harm in helping them out and switching automatically with a warning?

Reply:
I've changed the behaviour to just raise a ValueError and updated tests accordingly.